lib/controllers/notifications_controller.dart

```diff
@@ -42,28 +37,33 @@ class NotificationsController extends GetxController {
     return [];
   }
 
-  void scheduleNotificationForWeeklyClass(
-      String className, TimeOfDay classTime, int dayOfWeek) {
-    DateTime now = DateTime.now();
-    DateTime scheduledDate = DateTime(
-      now.year,
-      now.month,
-      now.day + (dayOfWeek - now.weekday + 7) % 7,
-      classTime.hour,
-      classTime.minute,
-    );
+  int _generateRandomIntId({int min = 100000, int max = 999999}) {
```

Comment:
Wouldn't it be better to create a finite know number of IDs incase you need to debug a notification?

Reply:
This was for testing purpose since I haven't implemented an interface yet for calling notifications but sure I intend to either have the IDs in a key value pair or an enum kind structure for the IDs

Reply:
Noted 👍🏾